### PR TITLE
Feature/archive nystagmus

### PR DESCRIPTION
**Pull request by ach206**
- removed nystagmus from being exposed for consumption

### CODE REVIEW

**Comment by ach206** [comment]
@bobbys-dev  is our `docs` directory being generated by swagger? I intentionally waited on updating those files inside `docs` in case there was an easier way to update (remove nystagmus from being mentioned)

**Comment by bobbys-dev** [comment]
> @bobbys-dev is our `docs` directory being generated by swagger? I intentionally waited on updating those files inside `docs` in case there was an easier way to update (remove nystagmus from being mentioned)

The ones in docs/ are generated by `typedoc` package, and they take information from the function signatures and modules. So you don't need to pick out files one by one to delete and instead you can delete docs/ entirely and then do `npx typedoc --out docs src/index.ts` to regenerate the docs from what's still there. FYI swagger deals with API documentation.

You'll also need to modify README.md and remove mentions of Nystagmus.

**Comment by bobbys-dev** [comment]
> @bobbys-dev is our `docs` directory being generated by swagger? I intentionally waited on updating those files inside `docs` in case there was an easier way to update (remove nystagmus from being mentioned)

@ach206 You'll need to delete docs/ and regenerate then using typedoc package, something like `npx typedoc --out docs src/index.ts`, but... since the Nystagmus modules still exist, I think you'll have to pass in a option to exclude that module. Not sure yet what exact string pattern to send to exclude Nystagmus-related props that will be generated, but I'll check it out myself too

**Comment by bobbys-dev** [comment]
@ach206 Ok disregard some of my last comment because I didn't pull this branch correctly, my fault. You can delete docs/ and then do `yarn run typedoc --out docs src --exclude "**/components/tasks/Nystagmus.tsx"`, and it should regenerate docs/ with all Nystagmus references  gone because you've already removed them.

**Review by bobbys-dev — APPROVED** [review]

**Review by bobbys-dev — APPROVED** [review]
Delete docs/ and regenerate  with `yarn run typedoc --out docs src --exclude "**/components/tasks/Nystagmus.tsx"`

File: example/src/screens/FullDemoScreen.tsx [hunk 1]
```diff
@@ -118,10 +118,10 @@ export default function FullDemoScreen() {
         A demonstration utilizing the various components provided in this SDK to
         replicate Senseye's data collection and processing workflow.
         {'\n\n'}
-        This demo will execute Calibration, Nystagmus, and PLR tasks. A video
-        will be recorded for each task and uploaded to Senseye servers for
-        processing. At the end, a result screen will display with an inference
-        of the results.
+        This demo will execute Calibration, and PLR tasks. A video will be
```
Comment (tam-nguyen-senseye) [inline example/src/screens/FullDemoScreen.tsx:121]: remove comma: "Calibration, "

**Comment by tam-nguyen-senseye** [comment]
> Delete docs/ and regenerate with `yarn run typedoc --out docs src --exclude "**/components/tasks/Nystagmus.tsx"`

my own preference, but i'd like to leave regenerating docs until we do a `release` or `hotfix` branch -- this prevents having the doc diffs drown out the meat of the changes

also, let's put this command into `package.json`:
```
"scripts": {
  ...
  "generate-docs":  "yarn run typedoc --out docs --json docs/senseye_react_sdk_docs.json src --exclude '**/components/tasks/Nystagmus.tsx'"
}
```

File: example/src/screens/NystagmusScreen.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ export default function NystagmusScreen() {
         onPress={() => setIsShowModal(true)}
       />
       <Modal visible={isShowModal} onRequestClose={() => setIsShowModal(false)}>
-        <Tasks.Nystagmus iterations={3} onEnd={() => setIsShowModal(false)} />
+        {/* <Tasks.Nystagmus iterations={3} onEnd={() => setIsShowModal(false)} /> */}
       </Modal>
```
Comment (Haplo-Dragon) [inline example/src/screens/NystagmusScreen.tsx:27]: Should we just be removing this file (`example/src/screens/NystagmusScreen.tsx`) entirely? Or is there a reason to keep it?

**Review by Haplo-Dragon — APPROVED** [review]

**Review by tam-nguyen-senseye — APPROVED** [review]

**Pull request merged**